### PR TITLE
New property for RHSSO config file

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc [hunk 1]
```diff
@@ -19,6 +19,15 @@ image::integration/kie_git_client_settings.png[]
 .. Click the *Installation* tab at the top of the client configuration screen and choose *Keycloak OIDC JSON* as a *Format Option*.
 .. Click *Download*.
 . Move the downloaded JSON file to an accessible directory in the server's file system or add it to the application class path.
+. In the `_EAP_HOME_/standalone/configuration/standalone-full.xml` file, under the `<system-properties>` tag, add the following system property:
++
+[source,xml]
+----
+<property name="rg.uberfire.ext.security.keycloak.keycloak-config-file" value="$EAP_HOME/kie-git.json"/>
```
Comment (RishiRajAnand) [inline doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc:26]: actual property name is '"org.uberfire.ext.security.keycloak.keycloak-config-file", we are missing an extra 'O' @mramendi

File: doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc [hunk 2]
```diff
@@ -19,6 +19,15 @@ image::integration/kie_git_client_settings.png[]
 .. Click the *Installation* tab at the top of the client configuration screen and choose *Keycloak OIDC JSON* as a *Format Option*.
 .. Click *Download*.
 . Move the downloaded JSON file to an accessible directory in the server's file system or add it to the application class path.
+. In the `_EAP_HOME_/standalone/configuration/standalone-full.xml` file, under the `<system-properties>` tag, add the following system property:
```
Comment (RishiRajAnand) [inline doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc:22]: Can we also mention in some way that this is an optional property and if not added default "$EAP_HOME/kie-git.json" is considered ? @mramendi

**Comment by tomasdavidorg** [comment]
image
This part should be removed. (User should set just the `org.uberfire.ext.security.keycloak.keycloak-config-file` property.)

**Review by tomasdavidorg — CHANGES_REQUESTED** [review]
I agree with @RishiRajAnand  comments + see my comment about the removing the old part.

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. Thanks 🙂

File: doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [id='sso-remote-services-securing-proc']
-= Securing {CENTRAL} file system services using RH-SSO
+= Enabling access to external file system and Git repository services for {CENTRAL} using RH-SSO
 
-To consume other remote services such as file systems (for example, a remote GIT service), you must specify the correct RH-SSO login module.
+To enable the {CENTRAL} to consume other remote services such as file systems and Git repositories using RH-SSO authentication, you must create a configuration file.
```
Comment (csherrar) [inline doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc:4]: [Style] I think adding a comma as below will help with the long sentence or perhaps the middle content can be separated?

To enable the {CENTRAL} to consume other remote services, such as file systems and Git repositories using RH-SSO authentication, you must create a configuration file.

File: doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [id='sso-remote-services-securing-proc']
-= Securing {CENTRAL} file system services using RH-SSO
+= Enabling access to external file system and Git repository services for {CENTRAL} using RH-SSO
```
Comment (csherrar) [inline doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc:2]: [Style] Depending on the relation between these, suggest that "system" is pluralized or the title rephrased if external services applies to file system and git repositories. Examples:

Enabling access to external file systems and Git repository services with {CENTRAL} by using RH-SSO
Enabling access to external services such as file systems and Git repositories by using RH-SSO
Accessing external file systems and Git repository services with {CENTRAL} by using RH-SSO

**Review by csherrar — APPROVED** [review]
Apart from a couple of suggestions, looks good to me.

**Review by RishiRajAnand — APPROVED** [review]

**Comment by RishiRajAnand** [comment]
Looks good now Thanks @mramendi

**Pull request merged**